chroma-bundles/chroma_support.repo

```diff
@@ -1,13 +1,3 @@
-[managerforlustre-manager-for-lustre]
-name=Copr repo for manager-for-lustre owned by managerforlustre
-baseurl=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/epel-7-$basearch/
-type=rpm-md
-skip_if_unavailable=True
-gpgcheck=1
-gpgkey=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/pubkey.gpg
-repo_gpgcheck=0
-enabled=1
-
```

Comment:
I think you should leave this in your patch.  If removing it fixes something then this is just papering over it and something else is still broken.